metaflow/datastore/task_datastore.py

```diff
@@ -462,7 +462,8 @@ def done(self):
             self._metadata.register_metadata(
                 self._run_id, self._step_name, self._task_id,
                 [MetaDatum(field='attempt-done', value=str(self._attempt),
-                           type='attempt-done', tags=[])])
+                           type='attempt-done',
+                           tags=['attempt_id:{0}'.format(self._attempt)])])
```

Comment:
what's the purpose of this?

Reply:
Just that we tag all metadata with attempt id and I guess this one fell through the cracks. I should have put it separately. This is for the UI mostly.